capstone/src/main/java/com/google/univiz/api/SuggestionDataApi.java

```diff
@@ -11,5 +11,5 @@
    * Takes a fragment of a college name or a complete college name and returns a list of
    * SuggestionData objects
    */
-  List<SuggestionData> getCollegeSuggestions(String collegeName);
+  List<SuggestionData> getSuggestions(String collegeName);
```

Comment:
Why is this changing in this PR?

Reply:
I will revert this change! I will be making another PR related to the SuggestionDataApi interface so I will include any other changes I want in that PR